packages/stylelint-config/package.json

```diff
@@ -11,6 +11,6 @@
   "license": "AGPL-3.0",
   "main": "index.js",
   "peerDependencies": {
-    "stylelint": "^14.9.1"
+    "stylelint": "^15.3.0"
```

Comment:
If we upgrade stylelint, we also need to take care of #10642.

Otherwise, please revert this change and upgrade it in a separate PR that addresses #10642.

This will affect everyone using this package so we do not want to provide default rulesets that cause a lot of deprecation warnings (or any other warnings / unnecessary console outputs either).